### PR TITLE
Create; ugc styles

### PR DESCRIPTION
**Pull request by chrisopedia**
Why is this pull request necessary:

1. provide some default styles for UGC/uncontrolled content

Changes proposed in this pull request:

- add in examples to showcase the various styles
- reset some basic styles to match default typographic styles

### CODE REVIEW

File: styles/helpers/ugc.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.ugc h1,
```
Comment (jgallen23) [inline styles/helpers/ugc.css]: why not nest?

File: examples/ugc/index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>UGC</title>
+    <link rel="stylesheet" href="../.dist/ugc.css"/>
```
Comment (jgallen23) [inline examples/ugc/index.html]: we might want to test against all of clientkit cause it loads the reset too

File: styles/helpers/ugc.css (hunk 1 above)
Comment (chrisopedia) [inline styles/helpers/ugc.css]: Didn't seem to be the norm in the rest of the files so I followed the convention.

File: examples/ugc/index.html (hunk 2 above)
Comment (chrisopedia) [inline examples/ugc/index.html]: K.  Good idea.

File: styles/helpers/ugc.css (hunk 1 above)
Comment (jgallen23) [inline styles/helpers/ugc.css]: lets nest!

File: styles/helpers/ugc.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
+.ugc {
+  & h1,
```
Comment (jgallen23) [inline styles/helpers/ugc.css]: you don't need &

**Pull request merged**